frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BookmarkList/BookmarkList.tsx

```diff
@@ -44,7 +44,7 @@ interface CollectionSidebarBookmarksProps {
   }: {
     newIndex: number;
     oldIndex: number;
-  }) => void;
+  }) => Promise<any>;
```

Comment:
In the future, it may be safer to use `Promise<void>` or something. Just incase someone tries to use whatever that promise may return